Scripts/UpdateRemoteDesktopSettings/Set-DTL_RemoteDesktopSettings.ps1

```diff
@@ -0,0 +1,90 @@
+﻿<#
+
+.SYNOPSIS
+This script leverages Azure Powershell to update the Remote Desktop settings in an Azure DevTest Lab.
+
+.DESCRIPTION
+The script updates the DevTest Lab's extended properties to include the experience level and/or the gateway settings.  When a user clicks "connect" on a Virtual Machine, the RDP file that's generated includes these settings.  In this way, IT admins or DevTest Lab Owners can configure the appropriate remote desktop settings for the environment and the uesrs can just leverage it seamlessly.
```

Comment:
Not a very big issue, but we may want to consider breaking this up so that it wraps to make it easier to read.

Reply:
Done!